src/universal/components/Dashboard/DashSearchControl.js

```diff
@@ -1,35 +1,54 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import withStyles from 'universal/styles/withStyles';
-import {css} from 'aphrodite-local-styles/no-important';
+import FontAwesome from 'react-fontawesome';
+import makePlaceholderStyles from 'universal/styles/helpers/makePlaceholderStyles';
+import ui from 'universal/styles/ui';
+import appTheme from 'universal/styles/theme/appTheme';
+import styled from 'react-emotion';
+
+const DashSearch = styled('div')({
+  alignItems: 'center',
+  display: 'flex',
+  flex: 1
+})
+
+const DashSearchIcon = styled(FontAwesome)({
+  color: appTheme.palette.dark60l,
+  marginRight: '.5rem'
+});
+
+const DashSearchInput = styled('input')({
+  appearance: 'none',
+  display: 'block',
+  border: 0,
+  fontSize: '.875rem',
+  outline: 'none',
+  padding: 0,
+  width: '100%',
+  ':focus': {
+    ...makePlaceholderStyles(ui.placeholderColorFocusActive)
+  },
+  ':active': {
+    ...makePlaceholderStyles(ui.placeholderColorFocusActive)
+  }
+});
 
 const DashSearchControl = (props) => {
-  const {onChange, placeholder, styles} = props;
+  const {onChange, placeholder} = props;
   return (
-    <input
-      className={css(styles.dashSearchInput)}
-      onChange={onChange}
-      placeholder={placeholder}
-    />
+    <DashSearch>
+      <DashSearchIcon name="search" />
+      <DashSearchInput
+        onChange={onChange}
+        placeholder={placeholder}
+      />
+    </DashSearch>
   );
 };
 
 DashSearchControl.propTypes = {
   onChange: PropTypes.func,
-  placeholder: PropTypes.string,
-  styles: PropTypes.object
+  placeholder: PropTypes.string
 };
 
-const styleThunk = () => ({
-  dashSearchInput: {
-    appearance: 'none',
-    display: 'block',
-    border: 0,
-    fontSize: '1.25rem',
-    outline: 'none',
-    padding: 0,
-    width: '100%'
-  }
-});
-
-export default withStyles(styleThunk)(DashSearchControl);
+export default DashSearchControl;
```

Comment:
@mattkrick oooooh, scrubbed that Aphrodite, lookie, lookie emotion-js

Reply:
@mattkrick does the markup for this component look right? Safe to drop `FontAwesome` as the param and compose a new component? Seems to work locally!

Reply:
yeah, that's _exactly_ how it should be used!
i tested it in prod & the styles still load correctly because the stylesheet that emotion creates for it always comes after the stylesheet for fontawesome. i can't imagine a scenario where this doesn't occur (i suppose the cdn goes down or replies slower than us?) so i'm comfortable using this pattern moving forward!